kuma/wiki/migrations/0001_initial.py

```diff
@@ -285,7 +284,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='document',
             name='team',
-            field=models.ForeignKey(blank=True, to='teamwork.Team', null=True),
+            field=models.ForeignKey(blank=True, to='wiki.DocumentTag', null=True),
```

Comment:
Seems like a hack? Do we plan to get rid of this in a later squashing of the migrations back into a single `0001` migration without referencing teamwork at all?